experimental/federated_plugin/federated_plugin/README.md

```diff
@@ -0,0 +1,14 @@
+# federated_plugin
+
+A new flutter plugin project.
```

Comment:
Could use a brief description here. Or just remove it entirely, since you have the main README in the root directory.